python/pylibcudf/pylibcudf/io/avro.pxd

```diff
@@ -20,4 +20,4 @@ cdef class AvroReaderOptionsBuilder:
     cpdef AvroReaderOptionsBuilder num_rows(self, size_type num_rows)
     cpdef AvroReaderOptions build(self)
 
-cpdef TableWithMetadata read_avro(AvroReaderOptions options)
+cpdef TableWithMetadata read_avro(AvroReaderOptions options, Stream stream = *)
```

Comment:
What does `= *` do in Cython? I don't think I've used this syntax but I have seen it in a few places.

Reply:
Indicates thhe argument is optional: https://docs.cython.org/en/latest/src/userguide/pyrex_differences.html#optional-arguments-in-cdef-cpdef-functions